packages/dialect-react-sdk/hooks/useThread.ts

```diff
@@ -78,11 +77,11 @@ const useThread = ({
 
   return {
     // sdk
-    thread,
+    thread: thread || null,
     delete: deleteThread,
 
     // react-lib
-    isFetchingThread,
+    isFetchingThread: thread === undefined && !errorFetchingThread,
```

Comment:
see comment above, wondering why it should be like this if it doesn't reflect what's really happens in the api

Reply:
It actually reflects the loading state what happens in API, better then isValidating. See SWR docs, 

https://swr.vercel.app/docs/getting-started#make-it-reusable.

The isFetching would be true every time the request is made, it will result into flickering loading state every pollingInterval. 

What we need instead just the initial loading.